config/resources/mailing-lists.json

```diff
@@ -0,0 +1,74 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "nco": "http://www.semanticdesktop.org/ontologies/2007/03/22/nco#",
+    "vcard": "http://www.w3.org/2006/vcard/ns#"
+  },
+  "resources": {
+    "contact-lists": {
+      "new-resource-base": "http://themis.vlaanderen.be/id/contact-lijst/",
+      "class": "nco:ContactList",
+      "attributes": {
+        "name": {
+          "type": "string",
+          "predicate": "vcard:fn"
+        }
+      },
+      "relationships": {
+        "contact-items": {
+          "predicate": "nco:containsContact",
+          "target": "contact-items",
+          "cardinality": "many"
+        }
+      },
+      "features": [
+        "include-uri"
+      ]
+    },
+    "contact-items": {
+      "new-resource-base": "http://themis.vlaanderen.be/id/contact-item/",
+      "class": "nco:Contact",
+      "attributes": {
+        "full-name": {
+          "type": "string",
+          "predicate": "vcard:fn"
+        },
+        "given-name": {
+          "type": "string",
+          "predicate": "vcard:given-name"
+        },
+        "family-name": {
+          "type": "string",
+          "predicate": "vcard:family-name"
+        },
+        "organization-name": {
+          "type": "string",
+          "predicate": "vcard:organization-name"
+        }
+      },
+      "relationships": {
+        "contact-lists": {
+          "predicate": "nco:ContactList",
+          "target": "contact-lists",
+          "cardinality": "many",
+          "inverse": true
+        },
+        "telephone": {
+          "predicate": "vcard:hasTelephone",
+          "target": "telephones",
+          "cardinality": "one",
+          "inverse": true
```

Comment:
Dit is niet de inverse relatie.  Het predicaat gaat van `nco:ContactList` naar `vcard:Voice`.
Je moet de inverse relatie ook toevoegen aan het model van `vcard:Voice`  (dus daar wel met `inverse: true`).

Zelfde comment voor e-mail.